plugins/cloudbuild/src/components/WorkflowRunsTable/WorkflowRunsTable.tsx

```diff
@@ -67,31 +68,31 @@ const generatedColumns: TableColumn[] = [
     title: 'Ref',
     render: (row: Partial<WorkflowRun>) => (
       <Typography variant="body2" noWrap>
-        <p>{row.substitutions?.BRANCH_NAME}</p>
+        {row.substitutions?.BRANCH_NAME}
       </Typography>
     ),
   },
   {
     title: 'Commit',
     render: (row: Partial<WorkflowRun>) => (
       <Typography variant="body2" noWrap>
-        <p>{row.substitutions?.SHORT_SHA}</p>
+        {row.substitutions?.SHORT_SHA}
       </Typography>
     ),
   },
   {
     title: 'Created',
     render: (row: Partial<WorkflowRun>) => (
-      <Typography variant="body2" noWrap>
-        <p>{moment(row.createTime).format('DD-MM-YYYY hh:mm:ss')}</p>
+      <Typography data-testid="cell-created" variant="body2" noWrap>
+        {DateTime.fromISO(row.createTime || '').toFormat('dd-MM-yyyy hh:mm:ss')}
```

Comment:
The `|| ''` here keeps it matching the behaviour of beforehand (which renders a now date).

Reply:
It's a little counter intuitive, and I'm not saying that the original behaviour is the right way either, just wondering if we can make this a little nicer by doing something like this?

```ts
DateTime.fromISO(row.createTime ?? DateTime.now()).toFormat('dd-MM-yyyy hh:mm:ss')
```

Reply:
@benjdlambert I agree that makes the intention much clearer :+1: Will update!